test/src/main/java/mil/nga/giat/geowave/test/HBaseStoreTestEnvironment.java

```diff
@@ -101,6 +160,43 @@ public void setup() {
 								conf)
 						.build();
 				hbaseLocalCluster.start();
+
+				if (enableVisibility) {
+
+					// Set valid visibilities for the vis IT
+					Connection conn = ConnectionPool.getInstance().getConnection(
+							zookeeper);
+					try {
+						SUPERUSER = User.createUserForTesting(
+								conf,
+								"admin",
+								new String[] {
+									"supergroup"
+								});
+
+						// List<SecurityCapability> capabilities =
+						// conn.getAdmin().getSecurityCapabilities();
+						// assertTrue(
+						// "CELL_VISIBILITY capability is missing",
+						// capabilities.contains(SecurityCapability.CELL_VISIBILITY));
+
+						// Set up valid visibilities for the user
+						addLabels(
+								conn.getConfiguration(),
+								auths,
+								User.getCurrent().getName());
+
+						// Verify hfile version
+						String hfileVersionStr = conn.getAdmin().getConfiguration().get(
+								"hfile.format.version");
+						assertTrue(
+								"HFile version is incorrect",
+								hfileVersionStr.equals("3"));
```

Comment:
this is less of a problem because we control the test environment, but still, maybe it'd be more flexible to make this >= 3?